internal/controllers/operator_controller_test.go

```diff
@@ -83,7 +83,7 @@ var _ = Describe("Operator Controller Test", func() {
 				By("running reconcile")
 				res, err := reconciler.Reconcile(ctx, ctrl.Request{NamespacedName: opKey})
 				Expect(res).To(Equal(ctrl.Result{}))
-				Expect(err).To(MatchError(fmt.Sprintf("no package '%s' found", pkgName)))
```

Comment:
Since I had similar comments on previous parts of this refactoring & most of our errors use double quotation marks I updated the format here.

I'm happy to revert this and move into a separate PR if this is too distracting

Reply:
+1 on keeping this